README.md

```diff
@@ -5,42 +5,66 @@
 
 <img src="icon.png" width="20%"/>
 
-Picoleto is the gentleman that will validate your Spanish identity documents.
+Picoleto is the gentleman that will validate your Spanish identity documents(NIF,NIE,Passport),Spanish Phones(MobileLine and LandLine) and Spanish ZipCodes.
```

Comment:
Maybe add a list of identity types here?

```markdown
Picoleto is the gentleman that will validate your Spanish documents:
* Identity 
    * NIF
    * NIE
    * Passport
* Phones
   * MobileLine
   * LandLine
* ZipCodes
```

Reply:
Sounds good @alorma ... what do you think @charlypro123?

Reply:
Yes,it become more understandable.

Reply:
Great, go ahead then! :tada: